### DISCUSSION TITLE
Multiple window support

### DISCUSSION THREAD

**Discussion by rdixitv** [minbrowser/min#1582] [Q&A]
Is it possible to get multiple windows in Min?

**Comment by blackgwe** [comment]
Maybe you can use task for it: https://minbrowser.org/tour/

**Comment by rdixitv** [comment]
Yeah, but I want to have two tabs side by side. Is there a way to do that?

**Reply by blackgwe** [reply]
>  two tabs side by side. Is there a way to do that?

I think it is not possible (yet).
